src/components/Anchor/Anchor.js

```diff
@@ -71,7 +75,9 @@ const inferType = (children) => {
   }
 
   return 'wrap';
-}
+};
+
+const isExternal = (to) => /^(https?)*:\/\//.test(to);
```

Comment:
It seems like this regex is just testing that a uri has `https`, what about `http` and `///_url_goes_here` links?